README.md

```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
 welcoming space for collaboration, and contributors are expected to adhere to
 the [code of conduct](https://github.com/panorama-ed/memo_wise/blob/main/CODE_OF_CONDUCT.md).
 
+## Releasing
+
+To make a new release of `MemoWise` to Rubygems, first install the release
+dependencies:
+
+```shell
+bundle config set --local with 'release'
```

Comment:
Do we need to undo this `bundle config set` afterwards?

Reply:
I would not think so? This just means that your local `bundle config` now installs the group `release` for this `Gemfile` -- what would be the benefit of undoing that?

Reply:
nvm, you're right.